team-policy.md

```diff
@@ -0,0 +1,138 @@
+
+Responsibilities:
+
+* Pays attention to the operation of the team
+* Pays attention to the team members
+* Reports wider context to the team
+
+### Facilitator
+
+* Current Holder: [@yaahc](https://github.com/yaahc)
+* Next Feedback Date: Wednesday 2023-02-01
+
+In order to be present with each other, we need a good facilitator .  Facilitators run meetings according to the format of meetings and decision making adopted by the group. Leader and facilitator are separate roles because facilitation and overseeing operations are different skill sets. They can be held by the same individual.
+
+Responsibilities:
+
+* Facilitates team meetings.
+* Pays attention to equivalence[^5] during meetings.
+* Supports planning of the agenda.
+* Distinguishes facilitator voice from team member voice.
+
+### Secretary
+
+* Current Holder: [@calebcartwright](https://github.com/calebcartwright)
+* Next Feedback Date: TODO
+
+In order to manage continuity with the team’s past, we need to have written records. The secretary manages the notes during the meeting, makes sure the minutes are distributed and accessible. The secretary also manages the records of the team and is the interpreter of policies.
+
+Responsibilities:
+
+* Makes sure meeting minutes are taken, approved, and stored.
+* Keeps track of all documents of the team.
+* Interprets the meeting minutes in case of disagreement.
+* Supports planning the agenda from the backlog.
+    * Tracking when policies and roles are due for review.
+
+## Meetings
+
+The style team meets weekly on wednesdays at 12:30pm PST [(everytimezone link)](https://everytimezone.com/s/3f88a253) for a weekly video sync. Agendas are posted in the [#T-style] zulip stream. 
+
+Unless otherwise noted, all of our meetings are public and open for anyone to attend. You will find the timing and event details on our [style team calendar](https://calendar.google.com/calendar/embed?src=d0564ed914a41cf4915bd5ebe6e2e4ec0ee1293fdc1d09d6f5bdb27d4f91c083%40group.calendar.google.com&ctz=America%2FLos_Angeles). We publish notes and minutes in written form in this github repository.
+
+### Agenda, Backlog, and Minutes
+
+The style team stores it's agendas, backlog, and meeting minutes in a single live [hackmd document](https://hackmd.io/@yaah/B1GZrzv4j) (rotating as necessary when the documents reach their length limit).
+
+The agenda should be prepared in advanced of each meeting by the facilitator with the assistance of the secretary and the lead. The lead is responsible for looking forward and adding agenda items such as new requests from other teams and users or new priorities and goals. The secretary is responsible for looking backwards and adding agenda items from existing work such as items from the backlog or policies that are due for review. The agenda proposal is then presented at the beginning of each meeting by the facilitator for the rest of the team to consent or object too.
```

Comment:
Not sure how literally we're intending to hold ourselves to this, but if we are planning on going word for word then I'd suggest we set an earlier mark for getting the agenda out. We've typically been running tight on time in past meetings and I think it'd be better to have the agenda set and generally agreed upon before the start of the meeting

Reply:
I tend to like to interpret things more literally. Happy to have the agenda proposal happen before meetings.

Reply:
Proposed before meetings yes; I think it's important for the agenda-consent process to happen in the meeting, though.